### PR TITLE
fix phpunit

### PR DESCRIPTION
**Pull request by folliked**
Phpunit was not running for a very long time :laughing: :rocket: :hocho: :rat:

### CODE REVIEW

**Comment by Nek-** [comment]
Oops, nice catch.

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="./tests/bootstrap.php" colors="true">
+<phpunit bootstrap="tests/bootstrap.php" colors="true">
     <testsuites>
         <testsuite name="Nekland Woketo">
-            <directory suffix="Test.php">./tests</directory>
```
Comment (Nek-) [inline phpunit.xml.dist]: The suffix was great as there are many other files in the test folder, isn't it ?

File: phpunit.xml.dist [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+            <directory>./tests</directory>
         </testsuite>
     </testsuites>
 
     <filter>
         <whitelist>
             <directory>./</directory>
             <exclude>
-                <directory>./Resources</directory>
```
Comment (Nek-) [inline phpunit.xml.dist]: 👍

File: phpunit.xml.dist (hunk 1 above)
Comment (folliked) [inline phpunit.xml.dist]: OFC :+1:

**Comment by folliked** [comment]
It seems good now :)

File: .travis.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ install:
     - composer install --prefer-source
 
 script:
-    - phpunit --coverage-clover=coverage.clover
+    - ./vendor/phpunit/phpunit/phpunit -c phpunit.xml.dist --coverage-clover=coverage.clover
```
Comment (valanz) [inline .travis.yml]: What about using `./bin/phpunit`?

Comment (folliked) [inline .travis.yml]: It does not change anything but ok

**Pull request merged**